maintainers/maintainer-list.nix

```diff
@@ -6880,6 +6991,7 @@
   };
   mattchrist = {
     email = "nixpkgs-matt@christ.systems";
+    matrix = "@mattchrist:matrix.org";
```

Comment:
I'm not active on Matrix and would rather not have this listed. If/when I am it will probably be at a different address.

Reply:
Sure, I've removed it now. Thank you.